docs/application/get-started/applications.md

```diff
@@ -4,9 +4,9 @@ Tizen is a user-interactive and service-oriented open source project that allows
 
 ## Tizen Profiles
 
-Tizen is built to work on a wide variety of platforms with a focus on embedded devices. In order to accommodate the various types of devices, a set of profiles has been defined to make it easier to develop applications for specific purposes and device types. Since Tizen 3.0, there are 3 profile types you can choose from: mobile, wearable, and TV. The mobile profile is designed for smart phones, the wearable profile is designed for smart watches, and the TV profile is designed for smart TVs.
+Tizen is built to work on a wide variety of platforms with a focus on embedded devices. In order to accommodate various types of devices, a set of profiles has been defined to make it easier to develop applications for specific purposes and device types. Since Tizen 3.0, it has traditionally supported 3 profile types: mobile, wearable, and TV. From Tizen 5.0, IoT is newly added to the list of profiles you can choose from. The mobile profile is designed for smart phones, the wearable profile is designed for smart watches, the TV profile is designed for smart TVs, and the IoT profile is desinged for IoT-based smart devices.
```

Comment:
Tizen is built to work on a wide variety of platforms with a focus on embedded devices. To accommodate various types of devices, a set of profiles has been defined to make it easier to develop applications for specific purposes and different device types. Since Tizen 3.0, Tizen supports three profile types, such as mobile, wearable, and TV. From Tizen 5.0, IoT is added to the list of profiles. The mobile profile is designed for smartphones, the wearable profile is designed for smartwatches, the TV profile is designed for smart TVs, and the IoT profile is designed for IoT-based smart devices.

@bson1012 

- Corrected the spelling mistake (desinged-->designed) 
- Kept most of the original text intact by suggesting minimalistic changes,

Reply:
@nawab-samsung 

Thanks for the comment.

I'd like to simplify some sentences from "Since Tizen 3.0, Tizen supports three profile types, such as mobile, wearable, and TV. From Tizen 5.0, IoT is added to the list of profiles. " to " Tizen supports four profile types, which are mobile, wearable, TV, and IoT."

Notice that I didn't use 'such as' to list four profile types as the list is complete itself. 
If it was a subset of a bigger list or some example from a list of more items, I would have used 'such as'.
However, for this case, I think using 'which are' would be more appropriate. 

Plz lemme know if you have any further concerns.
Thank you! :)

Reply:
@bson1012 
The current rephrase by you is as follows:

Tizen is built to work on a wide variety of platforms with a focus on embedded devices. To accommodate various types of devices, a set of profiles has been defined to make it easier to develop applications for specific purposes and different device types. Tizen supports four profile types, which are mobile, wearable, TV and IoT. The mobile profile is designed for smartphones, the wearable profile is designed for smartwatches, the TV profile is designed for smart TVs, and the IoT profile is designed for IoT-based smart devices.

You have missed a serial comma after TV 

Tizen supports four profile types, which are mobile, wearable, TV, and IoT. 

Kindly add a comma. 

For your information: 
We use **such as** to introduce an example or examples of something we mention. We normally use a comma before such as when we present a list of examples. Where there is just one example, we don’t need a comma. So, there is no question of the list being complete or not. 

(https://dictionary.cambridge.org/grammar/british-grammar/such-as)

Thanks!